packages/genesys-spark-components/src/components/stable/gux-pagination/gux-pagination.tsx

```diff
@@ -190,7 +193,12 @@ export class GuxPagination implements ComponentInterface {
 
   render(): JSX.Element {
     return (
-      <div class="gux-pagination-container">
+      <div
+        class={{
+          'gux-pagination-container': true,
+          'gux-disabled': this.disabled
```

Comment:
I would have expected you to pass `disabled` into `gux-pagination-items-per-page` and not have to add a class here.